core/storage/topic/gae_models.py

```diff
@@ -165,6 +167,27 @@ class TopicSummaryModel(base_models.BaseModel):
     version = ndb.IntegerProperty(required=True)
 
 
+class SubtopicPageModel(base_models.BaseModel):
+    """Subtopic model for an Oppia Topic.
+
+    This stores the html data for the subtopic landing page for all subtopics
+    throughout Oppia.
+
+    A SubtopicModel instance stores the following information:
+
+        id, topic_id, html_data, language_code.
+
+    The key of each instance is the topic id.
+    """
+
+    # The topic id that this subtopic is a part of.
```

Comment:
Any need to have a subtopic_id field? Not sure, just checking.

Reply:
If the topic_id is known, the subtopic_id can be deciphered from the subtopic_page id itself. So, I don't think it would be necessary.

Reply:
So, to confirm, you don't need to query by subtopic_id?

Reply:
Yeah, querying by subtopic id would be done only under a topic, right (in which case we have the actual model id)? I don't think we would be searching for all subtopics with a specific id across topics.

Reply:
Yep, that makes sense. Thanks!